glide.yaml

```diff
@@ -11,7 +11,7 @@ import:
   version: v1.0
 - package: github.com/lib/pq
 - package: github.com/mattn/go-sqlite3
-  version: v1.1.0
+  version: v1.2.0
```

Comment:
Updated this to see if it would fix build times but I just needed to do a proper package install.